### ISSUE TITLE
scylla_prepare: scylla_utils.py: YAMLLoadWarning: calling yaml.load() without Loader=...  is deprecated, as the default Loader is unsafe

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#4584]
*Installation details*
Scylla version (or git commit hash): 3.1.0.rc2-0.20190617.7d14514b8
Cluster size: 1 
OS (RHEL/CentOS/Ubuntu/AWS AMI): ami-0c2760b6168a02905

Create an AWS instance (i3.metal), and  check db log by : `journalctl  -u scylla-server -xe`

```
Jun 21 01:09:04 ip-172-10-244-29.ec2.internal scylla_prepare[12850]: /usr/lib/scylla/scylla_util.py:419: YAMLLoadWarning: calling yaml.lo
ad() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
Jun 21 01:09:04 ip-172-10-244-29.ec2.internal scylla_prepare[12850]: y = yaml.load(open(scylla_yaml_name))
Jun 21 01:09:11 ip-172-10-244-29.ec2.internal scylla[13078]: Scylla version 3.1.0.rc2-0.20190617.7d14514b8 starting ...
```

```
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 417) def get_scylla_dirs():
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 418)     """
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 419)     Returns a list of scylla directories configured in /etc/scylla/scylla.yaml
.
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 420)     Verifies that mandatory parameters are set.
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 421)     """
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 422)     scylla_yaml_name = '/etc/scylla/scylla.yaml'
0e47d8bb1dd (Vlad Zolotarov 2018-10-10 12:43:38 -0400 423)     y = yaml.load(open(scylla_yaml_name))
```

@avikivity @vladzcloudius

**Comment by amoskong** [comment]
This warning can be fixed by replace yaml.load() by yaml.safe_load()

```
[scylla]$ git grep yaml.load
dist/common/scripts/scylla_blocktune.py:    y = yaml.load(open(path))
dist/common/scripts/scylla_config_get.py:    cfg = yaml.load(s)
dist/common/scripts/scylla_fstrim:    cfg = yaml.load(s)
dist/common/scripts/scylla_io_setup:            cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))
dist/common/scripts/scylla_util.py:    y = yaml.load(open(scylla_yaml_name))
dist/docker/redhat/scyllasetup.py:        cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))
```

**Comment by vladzcloudius** [comment]
> This warning can be fixed by replace yaml.load() by yaml.safe_load()
> 
> ```
> [scylla]$ git grep yaml.load
> dist/common/scripts/scylla_blocktune.py:    y = yaml.load(open(path))
> dist/common/scripts/scylla_config_get.py:    cfg = yaml.load(s)
> dist/common/scripts/scylla_fstrim:    cfg = yaml.load(s)
> dist/common/scripts/scylla_io_setup:            cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))
> dist/common/scripts/scylla_util.py:    y = yaml.load(open(scylla_yaml_name))
> dist/docker/redhat/scyllasetup.py:        cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))
> ```

@amoskong Sounds like a good idea indeed. Please, send a patch.

**Comment by amoskong** [comment]
> > This warning can be fixed by replace yaml.load() by yaml.safe_load()
> > ```
> > [scylla]$ git grep yaml.load
> > dist/common/scripts/scylla_blocktune.py:    y = yaml.load(open(path))
> > dist/common/scripts/scylla_config_get.py:    cfg = yaml.load(s)
> > dist/common/scripts/scylla_fstrim:    cfg = yaml.load(s)
> > dist/common/scripts/scylla_io_setup:            cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))
> > dist/common/scripts/scylla_util.py:    y = yaml.load(open(scylla_yaml_name))
> > dist/docker/redhat/scyllasetup.py:        cfg = yaml.load(open(os.path.join(conf_dir, "scylla.yaml")))
> > ```
> 
> @amoskong Sounds like a good idea indeed. Please, send a patch.

Sent a patch to scylladb-dev list:
- [PATCH scylla] dist: suppress the yaml load warning

**Comment by slivne** [comment]
patch merged f0cd589a75a4e5c211b94a1e32aab8d90df32bc3
